File: study08/backend/springframework5/company/src/main/java/com/company/domain/Board.java
package com.company.domain;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Board {
    private int no;
    private String title;
    private String content;
    private String author;
    private String resdate;
    private int hits;
}

File: study08/backend/next/company/models/Board.js
// company/backend/models/board.js
const { DataTypes } = require('sequelize');
const sequelize = require('../config/database');

const Board = sequelize.define('Board', {
  no: { type: DataTypes.INTEGER, autoIncrement: true, primaryKey: true },
  title: { type: DataTypes.STRING, allowNull: false },
  content: { type: DataTypes.TEXT, allowNull: false },
  author: { type: DataTypes.STRING, allowNull: false },
  resdate: { type: DataTypes.DATE, defaultValue: DataTypes.NOW },
  hits: { type: DataTypes.INTEGER, defaultValue: 0 }
}, {
  tableName: 'board',
  timestamps: false
});

module.exports = Board;